src/db/util/init.ts

```diff
@@ -89,27 +82,3 @@ export async function initIdentity(db: DB): Promise<[UserDocument | null, Privat
 
   return [user, identity];
 }
-
-export async function initRemote(db: DB, identity: PrivateKey, rpcUrl: string) {
-  try {
-    if (!process.env.HUB_API_KEY || !process.env.HUB_API_SECRET) {
-      throw new Error('No Textile Hub credentials found');
-    }
-
-    const info: KeyInfo = {
-      key: process.env.HUB_API_KEY,
-      secret: process.env.HUB_API_SECRET,
-    };
-
-    const remote = await db.remote.setKeyInfo(info);
-
-    await remote.authorize(identity);
-
```

Comment:
this seems unrelated to the webpack change. let's at least document it in the PR description

Reply:
The @textile/hub library was throwing errors on compilation so I removed it. Another reason to get rid of textile soon is the bundle size.